client/src/components/header/header-main.js

```diff
@@ -0,0 +1,55 @@
+import Component from '@/components/component';
+
+import styles from '@/styles/components/header/header-main.module.scss';
+
+// props : { type, imageURL, onClick }
+export default class HeaderMain extends Component {
+  constructor(props) {
+    super(props);
+
+    this.$dom = this.createDom('header', {
+      className: styles['header'],
+    });
+
+    this.render();
+    this.addEvent();
+  }
+
+  render = () => {
+    this.$dom.innerHTML = `
+      <div class="${styles['left-area']}">
+        <button class="${styles['category-button']}">
+          <i class="wmi-category"></i>
+        </button>
+      </div>
+      <div class="${styles['middle-area']}">
+        <button class="${styles['place-button']}">
+          <i class="wmi-map-pin"></i>
+          <div class="${styles['place-name']}">${this._props.placeName}</div>
```

Comment:
```suggestion
          <div class="${styles['place-name']}">${this._props.location}</div>
```

뜬금 얘기이지만 _props는 잘못된 선택이였다는것 이제야 할게 되네요 ㅠ

Reply:
ㅠㅠ